src/components/Breadcrumb/index.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable import/no-extraneous-dependencies,no-script-url */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEllipsisH } from '@fortawesome/free-solid-svg-icons';
+import MenuItem from '../MenuItem';
+import ButtonMenu from '../ButtonMenu';
+import { uniqueId } from '../../libs/utils';
+import './styles.css';
+
+/**
+* An item in the hierarchy path of the page the user is on.
+* @category Layout
+*/
+export default function Breadcrumb(props) {
+    const {
+        href,
+        label,
+        onClick,
+        disabled,
+        overflowMenu,
```

Comment:
In the api doc I don't see this prop. This is not needed, if we need this then pass a ButtonMenu direct as children of the Breadcrumbs, but this case is not used commonly.